src/components/Filters/CollapsibleLabel.tsx

```diff
@@ -22,15 +34,15 @@ export interface CollapsibleLabelProps {
  *
  * @public
  */
-export function CollapsibleLabel({ label }: CollapsibleLabelProps): JSX.Element {
+export function CollapsibleLabel({ label, customCssClasses }: CollapsibleLabelProps): JSX.Element {
   const { isExpanded, getToggleProps } = useFilterGroupContext();
   const iconClassName = classNames('w-3 text-gray-400', {
     'transform rotate-180': !isExpanded
   });
 
   return (
     <button className='w-full flex justify-between items-center mb-4' {...getToggleProps()}>
-      <div className='text-neutral-dark text-sm font-medium text-left'>
+      <div className={twMerge('text-neutral-dark text-sm font-medium text-left', customCssClasses?.label)}>
```

Comment:
can we put the default styling in a `builtInCssClasses` variable like we do for other components? and we can use `useComposedCssClasses` instead of `twMerge` directly

Reply:
+2 on this!